src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEParameterSymbol.vb

```diff
@@ -51,6 +47,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         Private _lazyHasCallerMemberNameAttribute As ThreeState = ThreeState.Unknown
         Private _lazyHasCallerFilePathAttribute As ThreeState = ThreeState.Unknown
 
+        ''' <summary>
+        ''' The index of a CallerArgumentExpression. The value -2 means uninitialized, -1 means
+        ''' Not found. Otherwise, the index of the CallerArgumentExpression.
+        ''' </summary>
+        Private _lazyCallerArgumentExpressionParameterIndex As Integer = -2
```

Comment:
>-2 [](http://example.com/codeflow?start=73&length=2)

Consider making this a constant, and not -2 since we already use -2 for reduced extension method receivers.